flow-sample-basic/src/main/java/flow/sample/basic/HelloScreen.java

```diff
@@ -0,0 +1,47 @@
+package flow.sample.basic;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+final class HelloScreen implements Parcelable {
```

Comment:
In a follow up I'd be tempted to add auto parcel to the sample. I suspect most people aren't familiar with Parcelable and will wind up in the weeds.